packages/rest/src/types.ts

```diff
@@ -9,6 +9,22 @@ import {Request, Response} from 'express';
 
 export {Request, Response};
 
+/**
+ * Static assets settings
+ */
+export interface StaticAssetsConfig {
+  path: string;
+  rootDir: string;
+  options?: object;
+}
+
+export interface StaticComponent {
```

Comment:
The name `StaticComponent` does not make sense to me. It's really a component that contributes static assets. I propose to consider https://github.com/strongloop/loopback-next/pull/1924 so that we can give more flexibility to component providers. For example, the ExplorerComponent can expose a binding for its static assets so that the RestComponent can discover that and mount it.

Reply:
Agreed `StaticComponent` isn't the best name. We can decide on the appropriate name later, for now I chose something to get moving.